src/components/FilterForm.js

```diff
@@ -10,15 +10,33 @@ const Form = () => {
   );
 
   const [gender, setGender] = useState("M");
-  const [yearRange, setYearRange] = useState({ min: 1970, max: 1990 });
-  const [year, setYear] = useState(1900);
+  const [rangeToggle, setRangeToggle] = useState(false);
+  const [yearRange, setYearRange] = useState({ min: 1950, max: 1990 });
+  const [year, setYear] = useState(1950);
 
   const handleSubmit = () => {
-    let searchURL = `https://baby-namer-api.herokuapp.com/names?sex=${gender}&yearStart=${yearRange.min}&yearEnd=${yearRange.max}`;
+    let searchURL = `https://baby-namer-api.herokuapp.com/names?sex=${gender}&yearStart=${year}`;
+    if (rangeToggle) {
+      searchURL = `https://baby-namer-api.herokuapp.com/names?sex=${gender}&yearStart=${yearRange.min}&yearEnd=${yearRange.max}`;
+    }
     callList(searchURL);
     toggleDisplay();
   };
 
+  const toggleRange = () => {
+    // Activated before range state changes
+    if (!rangeToggle) {
+      if (yearRange.max <= year) {
+        setYearRange({ ...yearRange, max: year, min: yearRange.max });
+      } else {
+        setYearRange({ ...yearRange, min: year });
+      }
+    } else {
+      setYear(yearRange.min);
+    }
+    rangeToggle ? setRangeToggle(false) : setRangeToggle(true);
+  };
+
```

Comment:
Perform the logic based on `rangeToggle` here. Define a collection of variables that you can then use in the jsx below.